internal/subshell/sscommon/env.go

```diff
@@ -0,0 +1,49 @@
+package sscommon
+
+import "github.com/ActiveState/cli/internal/constants"
+
+type EnvType int
+
+const (
+	Deploy EnvType = iota
+	Default
```

Comment:
I don't see the point in making this an iota that then relies on a lookup table. The values could just be the envData.

Reply:
The idea was to restrict users of `WriteUserEnv` and `WriteRCFile` so they are not passing their own start, stop, and config key information that could potential conflict with deploy and default, or not have any identifying information at all. It's arguable if this package should dictate that though.

I've left a couple `var`s to be helpful but exported the data type.